### PR TITLE
Adds video api endpoints

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

**Comment by jbrundage** [comment]
@joshuaferr1s could you review?

I'll remove my change to the eslint config when I amend this PR with proper linting tomorrow after work

File: .env.sample [hunk 1]
```diff
@@ -2,5 +2,7 @@ NODE_ENV=development
 MONGO_URI=mongodb://localhost:27017/codinggardencommunity
 TEST_MONGO_URI=mongodb://localhost:27017/codinggardencommunity-test
 ADMIN_SECRET=keyboardcat
+YOUTUBE_API_KEY=aaaaaaaaaaaaaaaaaaaaaaaaaaa
+YOUTUBE_CHANNEL_ID=UCLNgu_OupwoeESgtab33CCw
```
Comment (ewenjo) [inline .env.sample:6]: The youtube channel ID can be hard coded, it doesn't have to be an environment variable.

File: src/tasks/seed/seedVideos.js [hunk 2]
```diff
@@ -0,0 +1,2286 @@
...
+    },
+    {
+        "type": "video",
+        "name": "Build a YouTube API Proxy Server with node.js/express and a Filterable List with Vanilla JavaScript",
+        "videoID": "Q4RiI7bvpso",
+        "title": "Build a YouTube API Proxy Server with node.js/express and a Filterable List with Vanilla JavaScript",
+        "date": "2018-05-18T04:14:44.000Z",
+        "description": "Show starts at 1:32 View the final product here: https://cg-videos.now.sh Code is here: https://github.com/CodingGarden/video-filter.",
+        "url": "https://www.youtube.com/watch?v=Q4RiI7bvpso",
+
```
Comment (ewenjo) [inline src/tasks/seed/seedVideos.js]: Async / Await

Comment (ewenjo) [inline src/tasks/seed/seedVideos.js]: Use template literals `https://www.youtube.com/watch?v=${item.id.videoId}`

Comment (ewenjo) [inline src/tasks/seed/seedVideos.js]: Object destructuring can be applied here.

**Review by ewenjo — COMMENTED** [review]
Just a few things, I will do a proper review after you've fixed the formatting and linting.

**Comment by ewenjo** [comment]
You have to upload the branch to the main repo before the tests can pass.

**Comment by jbrundage** [comment]
Closing in favor of #51

**Pull request closed without merging**